templates/security/livepatch.html

```diff
@@ -46,7 +46,7 @@ <h2 class="p-heading--four">Apply critical kernel patches without rebooting.</h2
     <div class="col-6">
       <h2>Free for personal use</h2>
       <p>All you need is an Ubuntu One account. Free for 3 machines.</p>
-      <p><a class="p-button--positive"  href="https://auth.livepatch.canonical.com/">Get Livepatch now</a></p>
+      <p><a class="p-button--positive"  href="https://ubuntu.com/advantage/">Get Livepatch now</a></p>
```

Comment:
```suggestion
      <p><a class="p-button--positive"  href="/advantage/">Get Livepatch now</a></p>
```
the link is local to this site.